tests/internal.EncoderBase.test.ts

```diff
@@ -66,15 +66,16 @@ describe('EncoderBase', () => {
       ).toStrictEqual({ a: 'a', b: 5, c: true })
     })
     test('sum', () => {
-      type SumType = 'A' | 'B' | 'C'
-      const sumType: SumType = 'A'
-      expect(
-        _.sum(sumType)({
-          A: _.record(_.literal('A')),
-          B: _.record(_.literal('B')),
-          // @ts-expect-error -- typelevel difference
-        }).encode({ A: 'A', B: 'B' })
-      ).toStrictEqual({ A: 'A', B: 'B' })
+      // TODO @jacob-alford 22-10-20: Figure out why Schemable2 / Encoder Sum is whacky
```

Comment:
There are type-level errors here, and I'm not totally sure why.  The tests pass fine, it just infers the input type as `never`